perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/ResourceExistsException.java

```diff
@@ -0,0 +1,23 @@
+package cz.metacentrum.perun.core.api.exceptions;
+
+/**
+ * Checked version of ResourceExistsException.
+ *
+ * @author Oliver Mrázik
+ */
+public class ResourceExistsException extends PerunException {
```

Comment:
It would be nice, if this exception could contain Resource param (set from constructor and with getter method). It would allow me to print better exception text in GUI. When necessary fill this with Resource object in conflict (already existing) so use can see, which resource is already there.